.github/workflows/ci.yml

```diff
@@ -16,17 +16,17 @@ jobs:
 
   test:
     needs: [lint]
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: true
```

Comment:
Why you removed tha `matrix:` ?

Reply:
Because to run mongodb tests, the Docker image on the CI only works with Linux machines.